index.html

```diff
@@ -5,7 +5,9 @@
 
     <title>Dash Budget Proposal Generator</title>
 
-    <link rel='shortcut icon' type='image/x-icon' href='favicon.ico' />
+		<link rel="icon" type="image/png" sizes="96x96" href="favicon-96x96.png">
```

Comment:
Probably just my OCD, but the whitespace in the front of the top and bottom lines here is bothering me. 😆

Reply:
Good catch, it was indented with tabs instead of spaces